enclave_wrangler/dataset_download.py

```diff
@@ -105,15 +107,13 @@ def datasets_views(datasetRid: str, file_parts: [str]) -> None:
                     raise e
                     parquet_parts.append(fname)
             else:
-                raise f'failed opening {url} with {response.status_code}: {response.content}'
+                raise RuntimeError(f'failed opening {url} with {response.status_code}: {response.content}')
```

Comment:
Was getting an error here in my IDE. So I added `RuntimeError()` which is what I use pretty often.